DangoPlop/Assets/Scripts/SpriteBasedLaser.cs

```diff
@@ -106,7 +106,7 @@ public class SpriteBasedLaser : MonoBehaviour
         /// The time between collision trigger events. If the laser is colliding with an object, this interval is used to fire the laser hit trigger event which other objects can subscribe to for custom functionality.
```

Comment:
You shouldn't be moving the script. Copy the contents of it. Leave the 2DLaserPack alone so we can check all the original code there when we want.